app/views/issues/_sidebar.html.erb

```diff
@@ -9,12 +9,12 @@
 
 <%= render 'shared/sidebar_collection',
   category_id: nil,
-  collection: @issues,
+  collection: @issues.sort,
```

Comment:
This is the controller's task, not the view's.

Reply:
fixed